### DISCUSSION TITLE
LINE-BOIL/SMOOTHING Tool!! 0_0 artists suffering from RSI will LOVE IT

### DISCUSSION THREAD

**Discussion by milkdestroyer** [keiwando/toonsquid#714] [Ideas]
Example (video): https://gyazo.com/f504a0edc4bef77b968fb0828bdcb068
Coming from an animator with tendonitis!!! I'd be over the MOON if Toonsquid could add a Lineboil feature!!! redrawing frames to get lineboil really kills me and shortens how long I can work in a session. This is the one and only feature missing from toonsquid that would make it perfect. It'd be great for those with sensitive hands and for those emulating the newgrounds/flash style :D

**Comment by DWrightAnimation** [comment]
Frankly, I'd be down for this feature being added, especially if someone using ToonSquid doesn't have an Apple Pencil, but wants to make their animations look hand drawn.

**Comment by thehenwithatie** [comment]
Made an account here to rally support as well! As someone who loves using lineboil in my animations, it would be super helpful to streamline the process!

**Comment by biblops** [comment]
I would kill for this feature. This one simple thing would take ToonSquid into platinum-tier status for me.
